src/MongoDB.Driver.Core/Core/Operations/RetryabilityHelper.cs

```diff
@@ -41,6 +43,11 @@ static RetryabilityHelper()
                 typeof(MongoCursorNotFoundException)
             };
 
+
+            __retryableReadExceptions = new HashSet<Type>(resumableAndRetryableExceptions)
+            {
```

Comment:
I guess, I see why it was done in this way, but it looks a little bit weird for me (i mean empty brackets).  Just to highlight, if you ok, I don't mind.